.github/workflows/formatting.yml

```diff
@@ -0,0 +1,50 @@
+name: Formatting
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/pr-fetch@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Download stanc3
+        run: |
+          cd ..
+          wget https://github.com/stan-dev/stanc3/releases/download/nightly/linux-stanc
+          chmod +x linux-stanc
+          cd helpful_stan_functions
+      
+      - name: Auto-format
+        run: |
+          for F in `find . -type f -name "*.stanfunctions"`
+          do
+            if [ $(grep "derivative_fun\|#include" $F | wc -l) -eq "0" ]; then
```

Comment:
We ignore files with includes and those that use `derivative_fun` function that does not exist. Not sure what to do for those.